src/transformers/__init__.py

```diff
@@ -4213,6 +4224,7 @@
         from .models.gpt2 import GPT2TokenizerFast
         from .models.gpt_neox import GPTNeoXTokenizerFast
         from .models.gpt_neox_japanese import GPTNeoXJapaneseTokenizer
+        from .models.gptsan_japanese import GPTSanJapaneseTokenizer
```

Comment:
Same here.

Reply:
Will it become impossible to use in this way?
```
>>> from transformers import GPTSanJapaneseTokenizer
>>> tokenizer = GPTSanJapaneseTokenizer.from_pretrained("Tanrei/GPTSAN-japanese")
```
It can be used by using AutoTokenizer or by importing from models.gptsan_japanese, so it's not really a problem, but I think it seems that many tokenizers can be used like that.

Reply:
I run `make fix-copies`, but any files are not changed...